docs/dashboards/custom-chart.md

```diff
@@ -0,0 +1,164 @@
+# Custom Chart
+
+The Custom Chartm Block is an advanced Block that allows you to create custom data visualization by utilizing [Vega-Lite](https://vega.github.io/vega-lite/). If you would like a specific visualization that's not already a Losant Dashboard Block, the Custom Chart Block gives you the ultimate support for a wide array of visualization types.
+
+![Data Table Scatter Plot](../images/workflows/custom-chart-data-table-scatter-plot.png "Data Table Scatter Plot")
+
+## Configuration
+
+Configuring the Custom Chart Block is broken up into three sections ...
+
+### Choosing an Application
+
+First, choose an application that owns the devices and/or the data tables you would like to query.
+
+![Select Application](/images/workflows/custom-chart-select-application.png "Select Application")
+
+### Datasets
+
+![Datasets](/images/workflows/custom-chart-datasets.png "Datasets")
+
+#### Time Series Datasets
```

Comment:
Yeah, These are called "Time Series Query" in the screenshot that you have. But, the header of this section is "Time Series Datasets". 

So, in the description you can make it clear what the distiction between a qurey and a dataset. 

### Time Series Query

A Time Series Query provides a dataset that represents state data from a single attribute from a selected device over a duration of time at a selected resolution.

Reply:
Could also add something here about how this is the same data that's used in the Time Series Block.